package.json

```diff
@@ -318,6 +318,20 @@
                                                         ],
                                                         "default": "String"
                                                     },
+                                                    "mode": {
+                                                        "type": "string",
+                                                        "description": "RTT operating mode.",
+                                                        "enum": [
+                                                            "NoBlockSkip",
+                                                            "NoBlockTrim",
+                                                            "BlockIfFull"
+                                                        ],
+                                                        "enumDescriptions": [
+                                                            "The target will add data to the channel only if it fits completely, otherwise it will skip the data.",
+                                                            "The target will add as much data to the channel as possible, without blocking.",
+                                                            "The target will block until there is enough space in the channel to add the data."
+                                                        ]
+                                                    },
                                                     "showTimestamps": {
                                                         "type": "boolean",
                                                         "default": false,
```

Comment:
@bugadani This default is out of synch with what is implemented on the probe-rs side, where you set default to true. Should we just omit the defaults here?